test-utils/src/latestDependencies/java/com/revenuecat/purchases/utils/stubsBillingClient4.kt

```diff
@@ -40,3 +42,32 @@ fun stubPurchaseHistoryRecord(
                 "purchaseToken": "$purchaseToken"
             }
         """.trimIndent(), signature)
+
+@SuppressWarnings("LongParameterList", "MagicNumber")
```

Comment:
deleted SkuDetailsStub to pull into here... i imagine we will have more differences in stubs soon